tests/conftest.py

```diff
@@ -7,6 +7,24 @@
 
 import tubular.base as base
 
+"""
+How To Use This Testing Framework
+-----------------------------------
+
+Fixtures in this file are used in tests for shared behaviour which can be run on multiple transformers through inheritance.
+setup_class is defined in each test class to define cls.transformer_name as the name of the transformer being tested.
+
+Test methods for shared behaviour are defined in base_tests.py or child test classes which are themselves inherited.  The
+fixtures in this file can be used in these tests to access initiated/ uninitiated transformers and minimum attributes for
```

Comment:
N/Q: I've called my fixtures (un)instantiated transformers, you have referenced them as them (un)initiated transformers. I'm not 100% sure what it should be, but I think we should be consistent. After looking around online I think we might be both wrong and want "(un)initialised".

As far as I understand it:
- Instantiation -> refers to creating a copy (instance) of an object, this is the work done by any constructor methods
- Initialisation -> passes arguments to the instance to set it up with an appropriate initial state (init)
 
I think the difference is more relevant in other OOP languages. Perhaps worth a look at whether a find and replace cleans this up?

https://realpython.com/python-class-constructor/

Reply:
simple with find and replace for instantiated to initialized: [ee8c4fa](https://github.com/lvgig/tubular/pull/192/commits/ee8c4fad11ddeedbcbbd42ef1488c06affcada92)

Reply:
cheers! :)